frontend/ui-pablo/app/components/Organisms/liquidity/RemoveForm/index.tsx

```diff
@@ -57,44 +57,39 @@ export const RemoveLiquidityForm: React.FC<BoxProps> = ({ ...rest }) => {
 
   useEffect(() => {
     if (poolId !== -1 && baseAsset && quoteAsset && parachainApi) {
-      const baseAssetId = baseAsset.supportedNetwork.picasso;
-      const quoteAssetId = quoteAsset.supportedNetwork.picasso;
-
-      if (baseAssetId && quoteAssetId) {
-        fetchSpotPrice(
-          parachainApi,
-          {
-            base: baseAssetId,
-            quote: quoteAssetId,
-          },
-          poolId
-        ).then((basePrice) => {
-          const basePriceBn = new BigNumber(basePrice);
+      fetchSpotPrice(
+        parachainApi,
+        {
+          base: baseAsset.network[DEFAULT_NETWORK_ID],
+          quote: quoteAsset.network[DEFAULT_NETWORK_ID],
+        },
+        poolId
+      ).then((basePrice) => {
+        const basePriceBn = new BigNumber(basePrice);
 
-          setPriceOfBase(basePriceBn);
-          setPriceOfQuote(new BigNumber(1).div(basePriceBn));
-        });
-      }
+        setPriceOfBase(basePriceBn);
+        setPriceOfQuote(new BigNumber(1).div(basePriceBn));
+      });
     }
   }, [poolId, baseAsset, quoteAsset, parachainApi]);
 
-  useEffect(() => {
-    if (parachainApi && debouncedPercentage > 0 && lpBalance.gt(0)) {
-      const selectedLpAmount = toChainUnits(
-        lpBalance.times(debouncedPercentage / 100)
-      );
-      (parachainApi.rpc as any).pablo
-        .redeemableAssetForGivenLpTokens(poolId, selectedLpAmount)
-        .then((response: any) => {
-          console.log(response);
-        }).catch((err: any) => {
-          console.error(err)
-        });
-    } else {
-      setExpectedRemoveAmountBase(new BigNumber(0));
-      setExpectedRemoveAmountQuote(new BigNumber(0));
-    }
-  }, [parachainApi, debouncedPercentage, lpBalance, poolId]);
+  // useEffect(() => {
+  //   if (parachainApi && debouncedPercentage > 0 && lpBalance.gt(0)) {
+  //     const selectedLpAmount = toChainUnits(
+  //       lpBalance.times(debouncedPercentage / 100)
+  //     );
+  //     (parachainApi.rpc as any).pablo
+  //       .redeemableAssetForGivenLpTokens(poolId, selectedLpAmount)
+  //       .then((response: any) => {
+  //         console.log(response);
+  //       }).catch((err: any) => {
+  //         console.error(err)
+  //       });
+  //   } else {
+  //     setExpectedRemoveAmountBase(new BigNumber(0));
+  //     setExpectedRemoveAmountQuote(new BigNumber(0));
+  //   }
+  // }, [parachainApi, debouncedPercentage, lpBalance, poolId]);
```

Comment:
Please remove commented code

Reply:
this is a WIP so it will be enabled later.